src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -2,6 +2,7 @@ package com.typesafe.sbt
 package packager
 package docker
 
+import java.io.File
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/nepomukseiler/sbt-native-packager/file/2358107371/issues/source?bid=2040925&fileBranchId=3219012#l5)